### DISCUSSION TITLE
公開済みの記事を編集している時に、公開記事と下書き記事の差分が見たい

### DISCUSSION THREAD

**Discussion by ko1nksm** [increments/qiita-discussions#197] [要望・アイデア・不具合]
公開済みの記事を意図せず編集してしまって、それに気が付かずに下書きに溜まっているのに気づいたときに「あれ？編集中になってる？これなにか書き換えてる最中だっけ？」となった時の回避策と言うのが私に「本当に必要だったもの」ではあるのですが、単純に公開している記事と下書きの差分がみれたら便利なのではないかと思います。

「#16 編集していない場合は下書きを保存しないでほしい」と少し関連した話になるかもしれません。例えば比較表示ページに削除ボタンがあれば、差分がない、または空白の違い程度の小さな違いであることがわかれば、安心して削除しやすくなるかなという意味です。

**Comment by kskwtnk** [comment]
@ko1nksm 
ご意見ありがとうございます！

現状、エディタには`差分`というタブがあり、ある程度はこちらで賄えるかと思っています 🙇 

| 本文タブ | 差分タブ |
| --- | --- |
| ![](https://user-images.githubusercontent.com/22670303/161663181-231e2fd1-a20f-4436-9a9f-1a5c4defe5c2.png) | ![](https://user-images.githubusercontent.com/22670303/161663226-b9e6c736-d552-44b1-bdd1-5ee09cb1ff8f.png) |

ただ、何も編集していない際は全文表示されてしまうなど、根本解決にはなっていないと思います
そのため #16 にもあるように、差分の無い記事は下書きに保存しないようにすることも検討中です 🙇
